pav-training/src/home/index.php

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="normalize.css">
+    <meta charset="utf-8">
+    <title>My web page</title>
+    <link rel="stylesheet" , href="/src/home/index.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="/src/home/index.js"></script>
+</head>
+
+<body>
+    <!-- The Header -->
+    <header>
+        <div>
+            <h4>The logo</h4>
+        </div>
+        <div>
+            <h2 class="slogan">The header slogan</h2>
+        </div>
+        <div id="form">
+            <ul>
+                <li>Hi<?php
+                        if (!isset($_SESSION['LOGGED_IN'])) {
+                        ?><span>Guest</span><?php
+                                        } else {
+                                            ?><span>pav</span><?php
+                                                            } ?>
+                </li>
+                <li><a href="javascript:void(0)" onclick="showLoginForm()">Login</a></li>
+            </ul>
+
+            <form id="login" method="post" action="/login">
+                <input type="text" name="username" placeholder="User name" value=<?php
+                                                                                    if (isset($_SESSION['rem_name'])) {
+                                                                                        echo $_SESSION['rem_name'];
+                                                                                    } ?>>
+                <input type="password" name="password" placeholder="Password" />
+                <label><input type="checkbox" name="rememberUsername" />Remember user name </label>
+                <button type="submit" name="Login">Login</button>
+                <form id="logout" method="post" action="/login">
```

Comment:
why should you need a form inside form but the same `action name` `login`? it makes us more confuse.

how about an <a> tag like 
```html
<a href="logout">Logout</a>
```